README.md

```diff
@@ -7,6 +7,7 @@ We utilize puppeteer to get chrome performance traces.
 
 To run all you have to do is start the server and run the test suite on it.
 ```
+npm run test-all

```

Comment:
I wanted it to be written in each separate block.
Because "test-all" and "test" are separate methods .